Kickstarter-iOS/DataSources/CommentsDataSource.swift

```diff
@@ -23,6 +23,8 @@ internal final class CommentsDataSource: ValueCellDataSource {
       return
     }
 
+    self.clearValues(section: Section.empty.rawValue)
```

Comment:
I think if we just update the first `clearValues` to `clearValues()` without passing a section in then it'll clear all of the values and we won't need two calls.